README.md

```diff
@@ -2,7 +2,7 @@
 A heartbeat plugin for LimeSurvey to ensure the session is kept alive.
 
 ## LimeSurvey support
-This plugin was built for LimeSurvey version 2.06 and does only support versions up to 2.50.
+This plugin was built for LimeSurvey version 2.06 and support versions up to 2.50 and 3.00
```

Comment:
Can't we just say up to 3.00 ?

Reply:
Yep …

Reply:
No, since it still work on 2.73 and 2.6lts :)

Reply:
Set to up than 2.6 , exception for 2.61